data/json/items/armor.json

```
@@ -10210,5 +10210,112 @@
     "material_thickness": 1,
     "environmental_protection": 1,
     "flags": [ "OVERSIZE", "OUTER", "WATERPROOF", "ALLOWS_NATURAL_ATTACKS" ]
+  },
+  {
+    "id": "bigback",
+    "type": "ARMOR",
+    "name": "high-volume rucksack",
+    "description": "Also know as \"a bigpack\", this rucksack is prized by pack rats but may lead to a bad back.",
+    "weight": 3539,
+    "volume": 85,
```

Rounding up a bit.
```suggestion
    "volume": "21 L",
```
True to the original
```suggestion
    "volume": "21250 ml",
```